conditional/templates/dashboard.html

```diff
@@ -251,15 +251,15 @@ <h3 class="panel-title">Major Projects</h3>
             </div>
             {% endif %}
 
-        </div>
+            {% if student and not active %}
+                <div class="alert alert-info"><span class="glyphicon glyphicon-exclamation-sign white" style="padding-right:5px"></span>
+                    Hey there! You're eligible to become an active member, click the button below if you'd like to pay dues and become active!
+                    <br />
+                    <a href="#" id="becomeActive" data-module="becomeActive" data-uid="{{username}}" class="btn btn-default">Become Active</a>
+                </div>
+            {% endif %}
 
-        {% if student and not active %}
-            <div class="alert alert-info"><span class="glyphicon glyphicon-exclamation-sign white" style="padding-right:5px"></span>
```

Comment:
Not sure an alert is the best container for this, I would go with a panel. 

If you really want to bring attention to it, a colored panel it definitely an option.

```
<div class="panel panel-info">
    <div class="panel-heading">
        <h3 class="panel-title">Panel info</h3>
    </div>
    <div class="panel-body">
        Panel content
    </div>
</div>
```